mappings/net/minecraft/entity/passive/FrogBrain.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS net/minecraft/unmapped/C_gtwuaauj net/minecraft/entity/passive/FrogBrain
+	FIELD f_ookmoifl LONG_JUMP_COOLDOWN_RANGE Lnet/minecraft/unmapped/C_hvilmtvi;
+	METHOD m_fhaglmeu addIdleActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_ihwohwid addLongJumpActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_ioifnvht (Lnet/minecraft/unmapped/C_gcfircge;)Z
+		ARG 0 frog
+	METHOD m_kbsketlv addCoreActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_kitbydka (Lnet/minecraft/unmapped/C_gcfircge;)V
+		ARG 0 frog
+	METHOD m_lixpcywj reset (Lnet/minecraft/unmapped/C_gcfircge;)V
+		ARG 0 frog
+	METHOD m_lvidokhl create (Lnet/minecraft/unmapped/C_rjqjaxef;)Lnet/minecraft/unmapped/C_rjqjaxef;
+		ARG 0 brain
+	METHOD m_lxhweavq addSwimActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_mtqywzuf addLaySpawnActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
```

Comment:
addLayTadpoleActivities or addTadpoleBirthActivities or addLayingActivities or addLayEggActivities there are tons of options here.

Reply:
I personally don't like addLaySpawn though, it's weird.

Reply:
the memory is called `LAY_SPAWN`, following that